soc/projects/numerics.md

```diff
@@ -82,6 +93,8 @@ and boundary cases.
 
 **Expected Results**: New and faster methods for evaluating properties of special functions.
 
+**Mentors:** [Alex Arslan](https://github.com/ararslan)
```

Comment:
Great! @ararslan there is no guiding documentation about this wrt to Julia yet but there is http://archive.flossmanuals.net/GSoCMentoring/what-makes-a-good-mentor and other pages in that manual which explain some responsibilities. I believe mentors get about $500 for their help.